centraldogma/build.gradle

```diff
@@ -8,10 +8,11 @@ ext {
 }
 
 dependencies {
-    compile project(":processor")
+    implementation project(":processor")
```

Comment:
In my understanding, "dependencies of library-X which required to compile a project that uses library-X's public APIs" should be appeared in compile classpath (i.e. `api` dependency)". Is that correct?

refs: (https://docs.gradle.org/current/userguide/java_library_plugin.html)
> dependencies required to compile against the published library (i.e. API dependencies)

Then I think this should be `api`? Because this public API refers `processor`'s class. (`PropertyDefinition`)
https://github.com/line/decaton/blob/master/centraldogma/src/main/java/com/linecorp/decaton/centraldogma/CentralDogmaPropertySupplier.java#L105

I guess same applies to `centraldogma-client`. (And also for other modules. (e.g. `processor`'s `kafka-clients`, `micrometer` etc))

Reply:
H-m. sounds like correct. let me fix that.